weasyl/controllers/user.py

```diff
@@ -139,12 +143,14 @@ def signin_2fa_auth_get_(request):
 
 @guest_required
 @token_checked
+@rate_limit_route(expiration_time=60 * 1)
 def signin_2fa_auth_post_(request):
     sess = define.get_weasyl_session()
+    referer = request.params["referer"] if 'referer' in request.params else "/"
```

Comment:
`request.params["referer"] if 'referer' in request.params else "/"` is `request.params.get("referer", "/")`